python/lsst/atmospec/spectraction.py

```diff
@@ -357,33 +372,40 @@ def run(self, exp, xpos, ypos, target, outputRoot=None, plotting=True):
         #     image, xpos, ypos = self.flipImageLeftRight(image, xpos, ypos)
         #     self.displayImage(image, centroid=(xpos, ypos))
 
-        # Use fast mode
         if parameters.CCD_REBIN > 1:
             self.log.info(f'Rebinning image with rebin of {parameters.CCD_REBIN}')
-            # TODO: Fix bug here where the passed parameter isn't used!
+            apply_rebinning_to_parameters()
             image.rebin()
             if parameters.DEBUG:
-                image.plot_image(scale='symlog', target_pixcoords=image.target_guess)
+                self.log.info('Parameters post-rebinning:')
+                dumpParameters()
+                self.debugPrintTargetCentroidValue(image)
+                title = 'Rebinned image with input target location'
+                image.plot_image(scale='symlog', target_pixcoords=image.target_guess, title=title)
+                self.log.debug('Post rebin:')
+                self.debugPrintTargetCentroidValue(image)
 
         # image turning and target finding - use LSST code instead?
         # and if not, at least test how the rotation code compares
         # this part of Spectractor is certainly slow at the very least
         if True:  # TODO: change this to be an option, at least for testing vs LSST
             self.log.info('Search for the target in the image...')
-            _ = find_target(image, image.target_guess)  # sets the image.target_pixcoords
+            # sets the image.target_pixcoords
+            _ = find_target(image, image.target_guess, widths=(parameters.XWINDOW, parameters.YWINDOW))
```

Comment:
Do you need to catch this at all?

Reply:
Probably not to be honest - I guess I just liked knowing that it was returning something, which we were ignoring, and chose to make that explicit. Is there a reason to drop that, or is that just a stylistic thing?

Reply:
That explanation is good enough to me for it to stay.